src/api/common/TutanotaConstants.ts

```diff
@@ -464,10 +464,10 @@ export const enum InputFieldType {
 export enum SecondFactorType {
 	u2f = "0",
 	totp = "1",
-	webauthn = "2",
+	webauthn = "2", // actually refers to u2f in client
 }
 
-export const SecondFactorTypeNames = ["U2F", "TOTP"]
+export const SecondFactorTypeNames = ["U2F", "TOTP", "U2F"]
```

Comment:
why is this in the client if it's not used in the client???

Reply:
Why are a lot of things in the client if they're not used in the client?

The entirety of SecondFactorTypeNames and InvoiceStatus is unused. getCertificateType is unused. CounterTypeToName is unused. PayPalData is unused. UsageTestState is unused.

Many of these things are not only unused, but they have likely *never* been used, but were kept in there for the admin client.

Reply:
I'm sorry, my frustration is not targeted at you but at whomever put it there/left it there.

For some things it did make sense that they are here (e.g. they are const enums and such) but this one should be in the admin client. If you agree, let's move it in there to avoid further breakage.

Reply:
Nah, I understand. This is incredibly frustrating for me to deal with, too. I think when people make a new enum that needs to be used outside of the server code, they sometimes default to putting it in TutanotaConstants.ts, since that's where all of the enums are.

And I agree! This should be moved.